Makefile

```diff
@@ -1,15 +1,16 @@
-GO_BUILD=GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags="-s -w"
-RELEASE_VERSION = "2.0.0"
-DOCKER_HUB_USER = "iguaziodocker"
+LOCATOR_TAG = 2.0.0
 
-all: lint bin image
+.PHONY: all
+all: lint build
 	@echo Done.
 
-bin: ensure-gopath
-	$(GO_BUILD) -o locatorctl cmd/locatorctl/main.go
+.PHONY: build
+build:
+	docker build --tag=$(LOCATOR_REPOSITORY)locator:$(LOCATOR_TAG) .
```

Comment:
This breakes the deployment if no one gonna pick it up correctly.
Please use a default value for local development: iguaziodocker or iguazio.

Reply:
Neither of those are valid. Dockerhub is not a thing anymore and there's no "primary" Docker registry